specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3261,6 +3261,105 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default": {
+      "get": {
+        "tags": [
+          "LineOfCredits"
+        ],
+        "x-ms-examples": {
+          "LineOfCreditBySubscription": {
+            "$ref": "./examples/LineOfCreditBySubscription.json"
+          }
+        },
+        "operationId": "LineOfCredits_Get",
+        "description": "Get the current line of credit.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/LineOfCredit"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default/increase": {
```

Comment:
It looks like you can declare a desired credit limit, not just increase it. Should the name of the action reflect adjust/modify not just increase?

Reply:
User cannot declare the credit limit through BillingRP. We are going to support only Increase scenario when user has existing credit limit. The declaration happens via some other system outside Billing RP.

Reply:
I totally understand they can't create a credit limit from the API but the way you have it defined, they could lower the limit not just increase it.

Reply:
Actually, we are internally calling the Commerce -> Credit Service API which allows only Increase.
They never allow to decrease the Credit Limit. Let's consider a scenario where user has a credit limit of $5000. When he requests to increase the limit to $2000 the request will be rejected.